lib/src/runner/plugin/platform_helpers.dart

```diff
@@ -108,7 +111,12 @@ Future<RunnerSuiteController> deserializeSuite(
       });
 
   return new RunnerSuiteController(
-      environment, suiteConfig, await completer.future,
+      environment,
+      suiteConfig,
+      await completer.future.timeout(_deserializeTimeout, onTimeout: () {
+        throw new ApplicationException(
+            "Timed out trying to deserialize the test suite.");
```

Comment:
I don't think "deserialize" is going to make a lot of sense to our users here. Maybe

```
"Timed out while loading the test suite.\n"
"It's likely that there's a missing import or syntax error."
```